site/_posts/2018-10-11-Announcing-Hammer-Beta1.html.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Announcing Hammer Beta Release
+author: jprause
+date: 2018-10-11 07:49:24 UTC
+comments: true
+published: true
+tags: releases announcements
+---
+
+On behalf of the ManageIQ team, we're pleased to announce the Beta release for ManageIQ Hammer! You can download the Hammer Beta Release [here](http://manageiq.org/download/)
+
+Hammer is the next milestone release for ManageIQ cloud and virtualization management platform. With each release, ManageIQ gets more robust and feature complete across providers. Here are just a few of the new or updated features.
```

Comment:
Maybe change "cloud and virtualization management" to "hybrid IT management" since we do so much more now,